Documentation/ColumnsConfig/Type/Slug.rst

```diff
@@ -23,9 +23,7 @@ With a URL like `https://www.typo3.org/ch/community/values/core-values/` a URL s
 Within TYPO3, a slug is always part of the URL "path" - it does not contain scheme, host, HTTP verb, etc.
 
 A slug is usually added to a TCA-based database table, containing some rules for evaluation and definition.
-
-In contrast to concepts within RealURL of "URL segments", a slug is a segment of a URL, but it is not limited
-to be separated by slashes. Therefore, a slug can contain slashes.
+A slug is a segment of a URL, but it is not limited to be separated by slashes. Therefore, a slug can contain slashes.
```

Comment:
> Therefore, a slug can contain slashes
... in a single path-segment.

Reply:
see #324

Reply:
@DavidBruchmann I am happy with any text as long as it does not say wrong things about RealURL :)

Reply:
@dmitryd nice that you can be satisfied so easy :)  
I changed it completely and it's already merged.